### PR TITLE
NEW : TakePOS - add buttons to allow keyboard usage for discounts

### PR DESCRIPTION
**Pull request by altairis-tof**
image

### CODE REVIEW

File: htdocs/takepos/reduction.php [hunk 1]
```diff
@@ -197,14 +202,33 @@ function ValidateReduction()
...
 		}
+		return false;
 	}
 </script>
 
 <div style="position:absolute; top:2%; left:5%; width:91%;">
 <center>
 <?php
-	print '<input type="text" class="takepospay" id="reduction_total" name="reduction_total" style="width: 50%;" placeholder="'.$langs->trans('Reduction').'">';
+	print '<input type="text" class="takepospay" id="reduction_total" name="reduction_total" size="10" placeholder="'.$langs->trans('Reduction').'">';
```
Comment (eldy) [inline htdocs/takepos/reduction.php:229]: Attribute "size" is deprecated in html5. Must use css or embedded style.

Comment (altairis-tof) [reply htdocs/takepos/reduction.php]: @eldy done; thanks for the autofocus that i did no know... time for me to update my skills or/and to hire younger developers than me ;)

**Comment by altairis-tof** [comment]
This PR is ready to merge

File: htdocs/takepos/reduction.php [hunk 2]
```diff
@@ -197,14 +202,33 @@ function ValidateReduction()
 			});
 		} else {
 			console.error('Error bad reduction type :', reductionType);
+			return false
+		}
+
+		return true;
+	}
+
+	$(document).ready(function() {
```
Comment (eldy) [inline htdocs/takepos/reduction.php:211]: Do you still need this now you added the "autofocus" property on the input field reduction_total ?

Comment (altairis-tof) [reply htdocs/takepos/reduction.php]: ok i remove the focus call; ready to merge.

**Pull request merged**